audit-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/audit/kafka/KafkaContractTest.java

```diff
@@ -139,221 +146,238 @@
 
     @BeforeEach
     void setup() {
-        errorBefore = Arrays.stream(new File(".").listFiles())
-                .filter(file -> file.getName().startsWith("Error"))
-                .collect(Collectors.toSet()).size();
-        successBefore = Arrays.stream(new File(".").listFiles())
-                .filter(file -> file.getName().startsWith("Success"))
-                .collect(Collectors.toSet()).size();
+
+        errorBefore = Arrays
```

Comment:
Only used once in a a test.  Would have expected to to be set-up in the one test instead of being set-up in the initialisation before each test

Reply:
Fixed that. Also fixed a couple of other potential bugs. The two file lists were looking in different places for the expected count and the actual count. I think it was just luck that the initial count was always 0. Also, the file were only getting cleared at the end of the test class and not per test method. That could have produced errors down the line. I think this solution is better now.